drivers/sensor/nxp/nxp_tmpsns/Kconfig

```diff
@@ -0,0 +1,13 @@
+# NXP i.MX RT temperature sensor configuration options
+
+# Copyright (c) 2025 Polytech A/S
+# SPDX-License-Identifier: Apache-2.0
+
+config NXP_TMPSNS
+	bool "NXP TMPSNS sensor"
+	depends on SENSOR
+	depends on DT_HAS_NXP_TMPSNS_ENABLED
+	depends on SOC_SERIES_IMXRT11XX || SOC_SERIES_IMXRT118X
```

Comment:
Do we need this `depends`. Is'nt is sufficient to base it on if the temperature sensor is enabled in device tree.

Reply:
Removed the dependency to SOC_SERIES_IMXRT